src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -24,12 +24,16 @@
     private static final String DOB_ATTRIBUTE_ID = "DOB";
     private static final Logger log = LoggerFactory.getLogger(ExamineeMapper.class);
 
-    private static final Map<String, String> attributeNames = ImmutableMap.of(
-        DOB_ATTRIBUTE_ID, "Birthday",
-        "Gender", "Sex",
-        "LastName", "LastOrSurname",
-        "SSID", "StudentIdentifier"
-    );
+    private static final Map<String, String> trtAttributeRelationshipTranslations = ImmutableMap.<String, String>builder()
+        .put(DOB_ATTRIBUTE_ID, "Birthday")
+        .put("Gender", "Sex")
+        .put("LastName", "LastOrSurname")
+        .put("SSID", "StudentIdentifier")
+        .put("DistrictID", "ResponsibleDistrictIdentifier")
```

Comment:
This came about because Nohemi noticed discrepancies between the TRT data that the legacy app was producing and the TRT that our new ERT is producing. There were a total of 8 keys (four relationships, and four attributes) that had different keys than what ART uses. Additionally we found that the date of birth needed to be formatted differently. Really this was the only driver.

However, looking into the actually TRT spec and the example XSD provided there - they are using the "old" keys - DistrictID and DOB for example.

Perhaps we should reach out to Brandt or someone at SBAC to get this cleared up? Sounds like either will work functionally (assuming these are aliased correctly), so I'm not sure if it matters much one way or another.